docs/ref/templates/builtins.txt

```diff
@@ -903,6 +903,25 @@ attribute and calling the result ``country_list``.
 * ``list`` -- a list of all items in this group (e.g., a list of all cities
   with country='India').
 
+.. versionchanged:: 1.11
```

Comment:
I added a commit to try and make the `versionchanged` annotation more self-contained. Is that better (if so, I can merge with the other commit)?